g/graylog2-server/graylog_master_rhel_8.3.sh

```diff
@@ -0,0 +1,66 @@
+# Package 	: graylog2-server
+# Version 	: master
+# Source repo 	: https://github.com/Graylog2/graylog2-server.git
+# Tested on 	: rhel_8.3
+# Maintainer 	: maniraj.deivendran@ibm.com
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# --------------------------------------------------------------------------------------
+# Note: Graylog2-server application had a dependency with testcontainers.
+# ====  Testcontainers communicate with other containers through docker.sock.
+#       Use options (--network="host" & -v /var/run/docker.sock:/var/run/docker.sock) if
+#       running the graylog2-server using docker container environment.
+# --------------------------------------------------------------------------------------
+#!/bin/bash
+
+VERSION=master
```

Comment:
I believe we should be building/testing this once against the latest stable release as well and if that works, specify that version here instead, because "master" will keep changing with time and it will be difficult to keep track of which code / which point we actually did the validation.

Reply:
Checkout master with specific commit id 20204a7979569aef116a1bdf30f4b9df6c52bc4d due to release 3.3.13/4.0.7 is unstable.